numpy/compat/py3k.py

```diff
@@ -91,9 +91,47 @@ def asunicode_nested(x):
     else:
         return asunicode(x)
 
-
 def is_pathlib_path(obj):
     """
     Check whether obj is a pathlib.Path object.
     """
     return Path is not None and isinstance(obj, Path)
+
+if sys.version_info[0] >= 3 and sys.version_info[1] >= 4:
+    def npy_import_module(name, fn, info=None):
+        """
+        Import and return module
+
+        name : str
```

Comment:
Needs the heading
```
Parameters
-----------------
```

See `doc/HOWTO_DOCUMENT.rst.txt` or other functions for docstring.